src/main/thrift/parquet.thrift

```diff
@@ -764,6 +845,14 @@ struct ColumnMetaData {
    * in a single I/O.
    */
   15: optional i32 bloom_filter_length;
+
+  /**
+   * Optional statistics to help estimate total memory when converted to in
+   * memory representations. The histogram contained on these statistics can
+   * also be useful in some cases for more fine-grained nullability/list length
+   * filter pushdown.
+   */
+  16: optional SizeStatistics size_estimate_statistics;
```

Comment:
```suggestion
  16: optional SizeStatistics size_statistics;
```

Should the name change for the field as well as the type?

Reply:
yes. done.